utils/db.ts

```diff
@@ -128,7 +120,8 @@ export async function deleteItem(item: Item) {
 }
 
 export async function getItem(id: string) {
-  return await getValue<Item>(["items", id]);
+  const itemsByIdKey = ["items", id];
+  return (await kv.get<Item>(itemsByIdKey)).value;
```

Comment:
```suggestion
  const res = await kv.get<Item>(["items", id]);
  return res.value;
```
This pattern is used in [the docs](https://deno.land/manual@v1.36.4/runtime/kv). Let's stick to that 🙂

Reply:
Gotcha